make/release.mk

```
@@ -15,6 +15,7 @@ release:
 								  ":(exclude)common/helm-repositories/ai-navigator-repos.yaml" \
 								  ":(exclude)services/ai-navigator-app" \
 								  ":(exclude)services/ai-navigator-cluster-info-agent"
+	./hack/hack/build-archive.sh $(ARCHIVE_NAME)
```

```suggestion
	./hack/build-archive.sh $(ARCHIVE_NAME)
```

Is there a duplicate `hack` in the path?